test/functional/readconcern_tests.js

```diff
@@ -164,35 +163,24 @@ describe('ReadConcern', function() {
           const succeeded = [];
           // Get a new instance
           const configuration = this.configuration;
-          let options;
-          let url;
-
-          if (test.urlReadConcernLevel || test.urlOptions) {
-            url = configuration.url();
-            if (test.urlOptions == null) {
-              url =
-                url.indexOf('?') !== -1
-                  ? `${url}&${test.urlReadConcernLevel}`
-                  : `${url}?${test.urlReadConcernLevel}`;
-            } else {
-              options = {
-                readConcern: {
-                  level: test.level
-                }
-              };
-            }
+
+          let url = configuration.url();
+          if (test.urlReadConcernLevel != null) {
```

Comment:
Since we're checking `urlReadConcernLevel != null` here and on L176, can we move client assignment up here?